### ISSUE TITLE
Cannot read property 'forEach' of null

### ISSUE THREAD

**Issue by lifeart** [ef4/ember-auto-import#36]
![image](https://user-images.githubusercontent.com/1360552/41798468-70eacdfa-7676-11e8-8e94-e03469b6459b.png)

```
   TypeError: Cannot read property 'forEach' of null
    at Object.keys.forEach.inPath (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\analyzer.js:144:23)
    at Array.forEach (<anonymous>)
    at groupModules (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\analyzer.js:143:24)
    at AnalyzerInternal.get imports [as imports] (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\analyzer.js:45:29)
    at Splitter.<anonymous> (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\splitter.js:29:42)
    at Generator.next (<anonymous>)
    at C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\splitter.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\splitter.js:3:12)
    at Splitter.depsForBundle (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\splitter.js:28:16)
              
                TypeError: Cannot read property 'forEach' of null
    at Object.keys.forEach.inPath (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\analyzer.js:144:23)
    at Array.forEach (<anonymous>)
    at groupModules (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\analyzer.js:143:24)
    at AnalyzerInternal.get imports [as imports] (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\analyzer.js:45:29)
    at Splitter.<anonymous> (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\splitter.js:29:42)
    at Generator.next (<anonymous>)
    at C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\splitter.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\splitter.js:3:12)
    at Splitter.depsForBundle (C:\Users\lifeart\Documents\Repositories\nextgen\node_modules\ember-auto-import\js\splitter.js:28:16)
              
```

**Comment by machty** [comment]
This happens for me when I delete a file (specifically a component.js file).

Log: https://gist.github.com/machty/3c2be3fe923d1ece647be107109c512c

**Comment by ef4** [comment]
Released in 1.0.0.
